service/view_building_coordinator.cc

```diff
@@ -305,11 +331,11 @@ future<> view_building_coordinator::send_task(view_building_target target, table
         co_return;
     }
 
-    int retries = 3;
+    int retries = RPC_RESPONSE_RETRIES_NUM;
```

Comment:
Please fix the issue in the commit which introduces it, and not in a later commit.